lib/ts/superTokens.tsx

```diff
@@ -108,52 +103,48 @@ export default class SuperTokens {
         });
     }
 
+    private async initMultitenancyWithDynamicLoginMethods(config: SuperTokensConfig) {
+        const multitenancyRecipe =
+            config.recipeList.find(({ recipeID }) => recipeID === Multitenancy.RECIPE_ID) ?? Multitenancy.init({});
+        const appInfo = normaliseInputAppInfoOrThrowError(config.appInfo);
+        const enableDebugLogs = config.enableDebugLogs === true;
+
+        // initialize auth-react and web-js multitenancy recipes
+        const multitenancyInstance = multitenancyRecipe.authReact(appInfo, enableDebugLogs) as Multitenancy;
+        multitenancyRecipe.webJS(appInfo, config.clientType, enableDebugLogs);
```

Comment:
This part is in sync, so it can be done in init. I know I boxed the entire thing in the last comment, but I only meant the async parts, the rest can stay there.
Also: aren't the init functions (`authReact` and `webJS`) called multiple times if the user added the MultiTenancy recipe to the recipe list?